PublicationsRouter/v2/api/MatchingConfig.md

```diff
@@ -0,0 +1,42 @@
+# Matching Config Structure
+Document contains structure of matching configuration JSON for both submission and retrieval. Used in POST/GET requests respectively against endpoint `/config`.
```

Comment:
This is too terse.  Also it suggested to me that there is a single structure, whereas there are 2 diff structs.